config/install/field.field.node.publication.field_publication_authors.yml

```diff
@@ -1,16 +1,19 @@
+uuid: 53e1711e-9e06-497f-8b29-7361d0bd0273
 langcode: en
 status: true
 dependencies:
   config:
     - field.storage.node.field_publication_authors
     - node.type.publication
-    - taxonomy.vocabulary.department
+    - taxonomy.vocabulary.organisation
+_core:
+  default_config_hash: '-ExNyXJOfyV6z82S2g3GreY6Vnqw4F48Y1fJ5BVTRWw'
```

Comment:
this one as well, remove it. That gets generated when it exported for a specific site.